### ISSUE TITLE
Foreground, entry point, and image tag

### ISSUE THREAD

**Issue by geozeke** [yrutschle/sslh#387]
Love the idea of moving docker image building to CI! Thanks, @oliv3r!

I've always built the image locally, and it's worked fine until recent commits. Here are two things I noticed:

1. The `README.md` file says to run the docker image from `ghcr.io/yrutschle/sslh:latest`, but the image seems to be packaged up as `ghcr.io/yrutschle/sslh:master`.
2. It doesn't matter if I clone the repo and manually build the image, or pull it from `ghcr.io/yrutschle/sslh:master`. It seems that the container starts normally, then immediately exits with a status code of `0`. If I roll back to `f23da1f`, and build manually, all is well. Any commit later than `f23da1f` doesn't seem to work.

Thanks for all the hard work on this!

**Comment by geozeke** [comment]
Sorry about that. This one's on me. I missed that the update requires the `--foreground` switch when you run the docker container. Updated the docker compose file and pulled directly from `ghrc.io` and all is well.

I think there's still a lingering documentation issue, though. The image that needs to be pulled is `ghcr.io/yrutschle/sslh:master`, but the `README.md` file says to pull this: `ghcr.io/yrutschle/sslh:latest`. If I've got it wrong, which is entirely possible, then recommend closing this issue. I'll just leave it up for now in case it's helpful to someone else.

**Comment by oliv3r** [comment]
Yeah, it's not so much a documentation issue, but lack of a proper tag :) once a tagged release is made, the `latest` tag will exist, until then, master. You still have to deal with the chicken-egg thing. Tag, update docs, or have docs ready, and wait for tag. :S

**Comment by geozeke** [comment]
👍
